src/layout/grid/gridColumn.jsx

```diff
@@ -1,119 +1,391 @@
+     * Deprecated prop. Please use `classes` to override styles.
+     */
+    desktopLarge: PropTypes.oneOf(['auto', true, false, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12]),
+    /**
+     * Deprecated prop. Please use `classes` to override styles.
+     */
+    fieldType: PropTypes.oneOf(['checkbox', 'legend', 'radio']),
+    /**
+     * Deprecated prop. Please use `classes` to override styles.
+     */
+    floated: PropTypes.oneOf(['left', 'right']),
+    /**
+     * Deprecated prop. Please use `classes` to override styles.
+     */
+    laptop: PropTypes.oneOf(['auto', true, false, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12]),
+    /**
+     * Deprecated prop. Please use `classes` to override styles.
+     */
+    mobile: PropTypes.oneOf(['auto', true, false, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12]),
+    /**
+     * Deprecated prop. Please use `classes` to override styles.
+     */
+    mobileLarge: PropTypes.oneOf(['auto', true, false, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12]),
+    /**
+     * Deprecated prop. Please use `classes` to override styles.
+     */
+    mobileMedium: PropTypes.oneOf(['auto', true, false, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12]),
+    /**
+     * Deprecated prop. Please use `classes` to override styles.
+     */
     style: PropTypes.shape({}),
-    tablet: PropTypes.oneOf(columnEnums),
-    textAlign: PropTypes.oneOf(textAlignEnums),
-    verticalAlign: PropTypes.oneOf(verticalAlignEnums),
-    width: PropTypes.oneOf(columnEnums),
+    /**
+     * Deprecated prop. Please use `classes` to override styles.
+     */
+    tablet: PropTypes.oneOf(['auto', true, false, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12]),
+    /**
+     * Deprecated prop. Please use `classes` to override styles.
+     */
+    textAlign: PropTypes.oneOf(['center', 'left', 'right']),
+    /**
+     * Deprecated prop. Please use `classes` to override styles.
+     */
+    verticalAlign: PropTypes.oneOf(['bottom', 'middle', 'top']),
+    /**
+     * Deprecated prop. Please use `classes` to override styles.
+     */
+    width: PropTypes.oneOf(['auto', true, false, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12]),
 };
 
 const defaultProps = {
+    classes: null,
+    children: null,
+    className: null,
+    id: null,
+    lg: false,
+    md: false,
+    sm: false,
+    xl: false,
+    // ****
+    // NOTE: All props below are deprecated and should not be used.
+    // ****
     align: undefined,
-    className: undefined,
-    desktop: undefined,
-    desktopLarge: undefined,
+    desktop: false,
+    desktopLarge: false,
     fieldType: undefined,
     floated: undefined,
-    id: undefined,
-    laptop: undefined,
-    mobile: undefined,
-    mobileLarge: undefined,
-    mobileMedium: undefined,
-    style: {},
-    tablet: undefined,
+    laptop: false,
+    mobile: false,
+    mobileLarge: false,
+    mobileMedium: false,
+    style: null,
+    tablet: false,
     textAlign: undefined,
     verticalAlign: undefined,
-    width: undefined,
+    width: false,
```

Comment:
Just curious: why is `false` a better default than `undefined`?

Reply:
This is a deprecated prop, but `false` provides a little more detail with what will happen with that specific breakpoint prop. It can either do true or false or 1-12.

Reply:
Okay ... I suppose.  `width={false}` (or `true` ... generally `width` as a Boolean) just seems odd to me.  But if you assure me it all makes sense ... all good.  Don't mind me.  :-)

Reply:
Resolving.